XenModel/Messages.resx

```diff
@@ -10458,7 +10458,7 @@ The master must be upgraded first, so if you skip the master, the rolling pool u
     <value>The current time on the pool master is: {0}</value>
   </data>
   <data name="SERVER_TOO_OLD" xml:space="preserve">
-    <value>This version of [XenCenter] supports [XenServer] 6.0 onwards.</value>
+    <value>This version of [XenCenter] supports [XenServer] [BRANDING_VERSION_6_0] onwards.</value>

```

Comment:
I don't think we have [BRANDING_VERSION_6_0] defined in branding. See re-branding.sh and 
Branding/branding.sh